controller/controller.go

```diff
@@ -732,6 +748,8 @@ func UpdateSchedulerConfig(
 				oldVersion,
 			)
 
+			// TODO: Send metrics of this error!!!
+			// this is the most critical part. If DBRollback fails, the watcher will keep spawning w
```

Comment:
spawning what?

Reply:
it was _spawning wrong gameserver_ but I will remove the comments as it does not makes sense anymore (I am sending metrics regarding this error and if it errors the watcher will perform a DBRollback)